### DISCUSSION TITLE
Take security serious, provide access to GitHub Advanced Security to all accounts

### DISCUSSION THREAD

**Discussion by yoshifp** [community/community#14389] [Enterprise]
Everyone should have the opportunity to access GitHub Advanced Security to ensure all software products are secure. I'm not asking for it to be free (it would be nice) but it should be accessible to all so all software can benefit and be more secure.

Currently this is limited to GitHub Enterprise customers and is then optionally enabled and Open Source projects.

Please make this feature available to all accounts.

**Comment by 6Bludawg9ine** [comment]
I have been reaching out for help for years now...you hear me...years..I deployed smart contracts and have watched my digital footprint be stolen...I've watched millions ..hundreds of millions of dollars be fraudulently moved ...I've authorized transaction after transaction from my device....that I wasn't aware of. I was placed in a digital market pipeline without my consent....I need...some...fucking...help....what the fuck.... PayPal coinbase,Google,cash app....I have the forensic timeline backed with sd cards...screenshots...system logs..showing what I am claiming...ease help email beallm789@gmail.com
